kubernetes/Vars

```diff
@@ -0,0 +1,4 @@
+DASHBOARD_DOMAIN=dashboard.openwisp.org
+CONTROLLER_DOMAIN=controller.openwisp.org
+RADIUS_DOMAIN=radius.openwisp.org
+TOPOLOGY_DOMAIN=topology.openwisp.org
```

Comment:
I am frankly not happy about the fact that even after making so many pull requests you have not added a blankline at the end! Please follow the reviews we make, they are really important to keep the project up and running. :cry:

Reply:
Sorry about that